### DISCUSSION TITLE
Numerical error in Delaunay triangulation

### DISCUSSION THREAD

**Discussion by tbirkandan** [pyvista/pyvista#4962] [Q&A]
We have been using PyVista to create a surface with `delaunay_2d()` as in the following function:

```
def createsurface(X,Y,Z):
    # Creates a PyVista surface using X,Y,Z coords. of points
    points=[list(i) for i in list(zip(X,Y,Z))]
    point_cloud = pv.PolyData(points)
    mysurface = point_cloud.delaunay_2d()
    return mysurface
```
 
Then we calculate the area of that surface using
 
`mysurface.area`
 
My question is how can we find the numerical error coming from i) the surface creation and, ii) area calculation?

**Comment by banesullivan** [comment]
This might be a a question better suited for the VTK discourse: https://discourse.vtk.org/
